### DISCUSSION TITLE
How to update the GDP data- I found laborproductivity

### DISCUSSION THREAD

**Discussion by Metondar** [JGCRI/gcam-core#294] [Q&A] [answered]
I am beginner in GCAM, and I tried to update the socioeconomic data using socioeconomic xml file. I updated the population, but the section of the GDP is using laborproductivity. How can I calculate those updates for the laborproductivity. I have the data for Population and GDP until 2070.

**Comment by hassaniazi** [comment, marked as answer]
Are you familiar with how to work with [gcamdata](https://jgcri.github.io/gcamdata/articles/getting-started/getting-started.html)? 

If yes, there is a socioeconomics file `\input\gcamdata\inst\extdata\socioeconomics\SSP_database_v9.csv` that has all the GDP information. You can directly change the country GDP there and rerun the data system to produce new `socioeconomics.xml`. 

If not, you can calculate Labor Productivity using the following expression 
`Labor Productivity = [ ((GDPy1/Populationy1) * (Populationyy0/GDPy0)) ^ (1/(y1 - y0) ] - 1`. Make sure you're using GDP MER (market exchange rate) for your calculations.

**Reply by Metondar** [reply]
Thank you, Dr., @Hassaniazi for your helpful response, it's greatly appreciated!
